File: AIPP course/Assignment 18/car_18(5).java
import java.util.*;

class Car {
    String brand, model;
    int year;

    Car(String brand, String model, int year) {
        this.brand = brand;
        this.model = model;
        this.year = year;
    }

    void displayDetails() {
        System.out.println("Car Details:");
        System.out.println("Brand: " + brand);
        System.out.println("Model: " + model);
        System.out.println("Year: " + year);
    }
}

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // Read input (CodeChef-compatible)
        String brand = sc.nextLine().trim();
        String model = sc.nextLine().trim();
        int year = Integer.parseInt(sc.nextLine().trim());

        // Create object
        Car car = new Car(brand, model, year);

        // Display details
        car.displayDetails();

        sc.close();
    }
}

File: AIPP course/Assignment 18/car_18(5).py
class Car:
    def __init__(self, brand, model, year):
        self.brand = brand
        self.model = model
        self.year = year

    def display_details(self):
        print("Car Details:")
        print(f"Brand: {self.brand}")
        print(f"Model: {self.model}")
        print(f"Year: {self.year}")

# Read input from stdin (CodeChef-friendly)
brand = input().strip()
model = input().strip()
year = input().strip()

# Create object
car = Car(brand, model, year)

# Display details
car.display_details()
